DelvUI/Interface/Jobs/ScholarHud.cs

```diff
@@ -197,18 +218,34 @@ public new static ScholarConfig DefaultConfig()
         [Checkbox("Fairy Gauge Text")]
         [CollapseWith(0, 2)]
         public bool ShowFairyText = true;
+               
+        [Checkbox("Seraph Bar Enabled")]
+        [CollapseWith(5, 2)]
+        public bool ShowSeraph = true;
+
+        [Checkbox("Seraph Bar Text")]
+        [CollapseWith(10, 2)]
+        public bool ShowSeraphText = true;
 
         [DragFloat2("Fairy Gauge Size", min = 1f, max = 2000f)]
-        [CollapseWith(5, 2)]
+        [CollapseWith(15, 2)]
         public Vector2 FairySize = new(254, 20);
 
         [DragFloat2("Fairy Gauge Position", min = -4000f, max = 4000f)]
-        [CollapseWith(10, 2)]
+        [CollapseWith(20, 2)]
         public Vector2 FairyPosition = new(0, -54);
 
         [ColorEdit4("Fairy Gauge Color")]
-        [CollapseWith(15, 2)]
-        public PluginConfigColor FairyColor = new(new Vector4(94f / 255f, 250f / 255f, 154f / 255f, 100f / 100f));
+        [CollapseWith(25, 2)]
+        public PluginConfigColor FairyColor = new(new Vector4(69f / 255f, 199 / 255f, 164f / 255f, 100f / 100f));
+
+        [ColorEdit4("Seraph Bar Color")]
+        [CollapseWith(30, 2)]
+        public PluginConfigColor SeraphColor = new(new Vector4(232f / 255f, 255f / 255f, 255f / 255f, 100f / 100f));
+                
+        //[ColorEdit4("Aetherpact Color")]
+        //[CollapseWith(35, 2)]
+        //public PluginConfigColor AetherpactColor = new(new Vector4(255f / 255f, 255f / 255f, 155f / 255f, 100f / 100f));
```

Comment:
Please remove if this is not going to be used

Reply:
Oh yeah I forgot about that, fixed.